components/emoji_picker/emoji_picker_tabs.jsx

```diff
@@ -70,6 +70,13 @@ export default class EmojiPickerTabs extends PureComponent {
                     bottom: this.props.style.bottom,
                     right: this.props.rightOffset,
                 };
+            } else if (this.props.placement === 'right') {
+                // Placement right is only for the emoji overlay in the custom status modal and this styling is specific for that
+                pickerStyle = {
+                    ...this.props.style,
+                    left: this.props.style.left + 3,
+                    top: this.props.style.top + 3,
```

Comment:
Why don't we pass the correct `top` and `left` values instead of applying a correction here?